### PR TITLE
Adding deepnet cross-validations

### PR DESCRIPTION
**Pull request by mmerce**

### CODE REVIEW

File: cross-validation/deepnet/script.whizzml [hunk 1]
```diff
@@ -0,0 +1,517 @@
...
+;; Output: (map) options map
+;;
+(define (get_deepnet_options batch_normalization
+                             default-numeric-value
+                             dropout-rate
+                             hidden-layers
+                             learn-residuals
+                             learning-rate
+                             max-iterations
+                             max-training_time
```
Comment (akashenfelter) [inline cross-validation/deepnet/script.whizzml]: Should be "max-training-time"

**Comment by akashenfelter** [comment]
When running the test, I'm getting an error when testing the deepnet script.
`Failed to obtain a finished resource:
{u'code': -1, u'call_stack': [[0, [60, 60], [5, 60]], [0, [496, 496], [4, 64]], [0, [500, 517], [2, 32]]], u'source_location': {u'origin': 0, u'lines': [60, 60], u'columns': [5, 60]}, u'elapsed_times': {u'in-progress': 77268, u'started': 127, u'queued': 50}, u'instruction': {u'source': {u'origin': 0, u'lines': [60, 60], u'columns': [5, 60]}, u'instruction': u'apply'}, u'elapsed': 77395, u'error': -8200, u'progress': 0.1, u'message': u'Problem while executing script: Error handling resource (Validation error)', u'cause': {u'code': -1206, u'http_status': 400, u'extra': {u'evaluations': u'Cross-validation is available to evaluations for models, ensembles, and logistic regression: 59e7af294006831a1800799c'}}}.`

More specifically, I think this comes from the function `create-k-models`. While is seems to work fine for other resource types, using "type" as "deepnet" returns the error `Error computing primitive operation 'create'. Expected arguments of types [[resource-type [resource-id map]] [resource-type resource-id [resource-id map]] [resource-type resource-id resource-id map]], but got ["deepnet" {"datasets" ["dataset/59e7a78b7e0a8d0cba000faa" "dataset/59e7a78c3645270f7400118...]`. I'm not sure exactly why, but it seems we can't create a list of deepnets from a list of datasets as we can with other resources.

**Comment by mmerce** [comment]
Yes, we need to wait for the apian version in next. They have extended the list of models to include deepnets.

File: cross-validation/deepnet/script.whizzml (hunk 1 above)
Comment (mmerce) [reply cross-validation/deepnet/script.whizzml]: Ooops, thanks!

**Comment by akashenfelter** [comment]
Tests are finally passing, apain must be ready! Merging.

**Pull request merged**